app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,53 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end
+
+  def show
+    find_question
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def edit
+    find_question
+  end
+
+  def create
+    @question = Question.new(question_params)
+
+    if @question.save
+      redirect_to @question, notice: 'Question was successfully created'
+    else
+      render :new
+    end
+  end
+
+  def update
+    find_question
+
+    if @question.update(question_params)
+      redirect_to @question, notice: 'Question was successfully updated'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.